boxes.yaml.example

```diff
@@ -1,11 +1,9 @@
 ---
 centos7-devel:
   box: centos7
-  shell: 'yum -y install ruby && cd /vagrant && ./setup.rb'
-  options: --scenario=katello-devel
-  installer: --katello-devel-github-username <my_github_username>
   ansible:
-    group: 'server'
+    playbook: 'playbooks/katello_devel.yml'
+    group: 'devel'
```

Comment:
Also maybe more obvious with github in the variable name, to match the installer flag